clients/shared/default_value.go

```diff
@@ -58,8 +57,7 @@ func BackfillColumn(dwh destination.DataWarehouse, column columns.Column, tableI
 		return nil
 	}
 
-	if _, ok := dwh.Dialect().(mssqlDialect.MSSQLDialect); ok {
-		// TODO: Support MSSQL column backfill
+	if !dwh.SupportDefaultValue() {
```

Comment:
In this case, should we continue backfilling with default values the old way? And only alter the column default (in https://github.com/artie-labs/transfer/pull/885) if this is true?

Reply:
I think we should get rid of our backfill completely because it's error prone